todolist/client/src/main/scala/handlers/TagClientHandler.scala

```diff
@@ -24,6 +24,7 @@ import examples.todolist.client.clients.TagClient
 import examples.todolist.protocol._
 import examples.todolist.protocol.Protocols._
 import higherkindness.mu.rpc.protocol.Empty
+// will need to address this
 import freestyle.tagless.logging.LoggingM
```

Comment:
it looks like many of our existing handlers are relying on freestyle for logging; so in addition to updating our reliance on the todo-list lib for handling the service logic and algebras, we will need to update our reliance on tagless logging.  I imagine we can do something similar to what we do in `PingPongClientHandler` and just use log4s.

Reply:
As discussed offline, yes, let's remove the dependency on freestyle for logging. I think some examples use log4s and others use log4cats. Either one should be fine.

Reply:
Yup, sounds good!  I've used log4s before in the past so I might just go with that because it's familiar and comfy.

Reply:
Ended up going with log4cats because it plays nicer with our cats-heavy codebase.  Made this change in https://github.com/higherkindness/mu-scala-examples/pull/11; will try and has this PR focus mostly on rewriting the todolist-lib to decouple from freestyle.